Nightlies are currently broken

```
---- [run-make] run-make/llvm-pass stdout ----
	
error: make failed
status: exit code: 2
command: "make"
stdout:
------------------------------------------
make[2]: Entering directory `/buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/obj/tmp/distcheck/rustc-nightly/src/test/run-make/llvm-pass'
g++ -Wall -Werror -g -fPIC -m64   -c llvm-function-pass.so.cc -o /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/obj/tmp/distcheck/srccheck/x86_64-unknown-linux-gnu/test/run-make/llvm-pass.stage2-x86_64-unknown-linux-gnu/libllvm-function-pass.o
make[2]: Leaving directory `/buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/obj/tmp/distcheck/rustc-nightly/src/test/run-make/llvm-pass'

------------------------------------------
stderr:
------------------------------------------
llvm-function-pass.so.cc:15:23: fatal error: llvm/Pass.h: No such file or directory
compilation terminated.
make[2]: *** [/buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/obj/tmp/distcheck/srccheck/x86_64-unknown-linux-gnu/test/run-make/llvm-pass.stage2-x86_64-unknown-linux-gnu/libllvm-function-pass.o] Error 1

------------------------------------------

thread '[run-make] run-make/llvm-pass' panicked at 'explicit panic', /buildslave/rust-buildbot/slave/nightly-dist-rustc-linux/build/obj/tmp/distcheck/rustc-nightly/src/tools/compiletest/src/runtest.rs:1595
note: Run with `RUST_BACKTRACE=1` for a backtrace.


failures:
    [run-make] run-make/llvm-pass
```

This is likely related to a mixture of cross compilation and https://github.com/rust-lang/rust/pull/33093

--- a/mk/tests.mk
+++ b/mk/tests.mk
@@ -636,11 +636,11 @@ CTEST_COMMON_ARGS$(1)-T-$(2)-H-$(3) := \
         --host-rustcflags "$(RUSTC_FLAGS_$(3)) $$(CTEST_RUSTC_FLAGS) -L $$(RT_OUTPUT_DIR_$(3))" \
         --lldb-python-dir=$(CFG_LLDB_PYTHON_DIR) \
         --target-rustcflags "$(RUSTC_FLAGS_$(2)) $$(CTEST_RUSTC_FLAGS) -L $$(RT_OUTPUT_DIR_$(2))" \
-	--cc '$$(CC_$(3))' \
-	--cxx '$$(CXX_$(3))' \
-	--cflags "$$(CFG_GCCISH_CFLAGS_$(3))" \
-	--llvm-components "$$(LLVM_ALL_COMPONENTS_$(3))" \
-	--llvm-cxxflags "$$(LLVM_CXXFLAGS_$(3))" \
+	--cc '$$(CC_$(2))' \
+	--cxx '$$(CXX_$(2))' \
+	--cflags "$$(CFG_GCCISH_CFLAGS_$(2))" \
+	--llvm-components "$$(LLVM_ALL_COMPONENTS_$(2))" \
+	--llvm-cxxflags "$$(LLVM_CXXFLAGS_$(2))" \
         $$(CTEST_TESTARGS)
 
 ifdef CFG_VALGRIND_RPASS

--- a/src/tools/compiletest/src/runtest.rs
+++ b/src/tools/compiletest/src/runtest.rs
@@ -2063,6 +2063,11 @@ fn run_incremental_test(config: &Config, props: &TestProps, testpaths: &TestPath
 }
 
 fn run_rmake_test(config: &Config, _props: &TestProps, testpaths: &TestPaths) {
+    // FIXME(#11094): we should fix these tests
+    if config.host != config.target {
+        return
+    }
+
     let cwd = env::current_dir().unwrap();
     let src_root = config.src_base.parent().unwrap().parent().unwrap()
                                   .parent().unwrap();

